app/services/tenor.js

```diff
@@ -94,9 +98,10 @@ export default class TenorService extends Service {
 
     @task({group: 'loadingTasks'})
     *loadTrendingTask() {
-        this.loadedType = 'trending';
+        this.loadedType = 'search';
 
         yield this._makeRequest(this.loadedType, {params: {
+            q: 'excited',
```

Comment:
It's also not clear where the change was made here from a "trending" query to an "excited" search.

Reply:
In the previous API definition, it allowed to pass a empty query, and the endpoint would receive the "trending" parameter. This would return the trending gifs in the app. This functionality has been removed, instead, the new documentation indicates that there must be a search done on each request, we can no longer send an empty query, neither can we request for the "trending" gifs.

In the new documentation (https://tenor.com/gifapi/documentation) they make examples querying for "excited" to show "Happy" and "Excited" gifs. I've took that example and implemented it here, so when the user loads the Tenor card, it'll load with "excited" gifs.

If we don't update the loadedType to `search` it will provoke a 404 error since that endpoint no longer exists.
Also, if we use 'search" as loadedType but we don't send parameter, it will return a 400 error.